test/packages/knowledge_base/manifest.yml

```diff
@@ -0,0 +1,27 @@
+format_version: 3.4.0
+name: knowledge_base
+title: Knowledge base test content package
+description: >
+  This package is a dummy example for packages containing a knowledge base entry.
+version: 0.1.0
+type: content
+source:
+  license: "Elastic-2.0"
+conditions:
+  kibana:
+    version: '^8.15.0'
```

Comment:
Are knowledge bases going to be available in all kinds of deployments? (in all serverless project types, on-prem and cloud)
Wondering in case we need to add a [capability](https://github.com/elastic/package-spec/blob/d9888bd70ae1d344a98ffc1dae08eccc1f90ad63/spec/integration/manifest.spec.yml#L117), we might need to add one if this feature is not going to be available in all kinds of deployments.

Reply:
Yeah, they will be available on all kinds of deployments